schema/mysql/v8/visibility/schema.sql

```diff
@@ -85,6 +86,7 @@ CREATE INDEX by_temporal_namespace_division   ON executions_visibility (namespac
 CREATE TABLE custom_search_attributes (
   namespace_id      CHAR(64)  NOT NULL,
   run_id            CHAR(64)  NOT NULL,
+  version           BIGINT    NOT NULL DEFAULT 0,
```

Comment:
I'm uncertain we need this. It would hinge on the way a write returns in MySQL. I'm fairly sure that currently if the proposed version is smaller than the written version we won't write, but also won't error. This needs to be addressed.

Reply:
I think you need this.

Reply:
Yea, particularly when I consider that I think our query will return with 1 row written no matter what happens with version. 

And now that I think of it, we could have an awful situation were one table is ahead and the other isn't. Yuck.